src/frequenz/client/common/metric/__init__.py

```diff
@@ -0,0 +1,141 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Module to define the metrics used with the common client."""
+
+from __future__ import annotations  # required for constructor type hinting
+
+from enum import Enum
+
+# pylint: disable=no-name-in-module
+from frequenz.api.common.v1.metrics.metric_sample_pb2 import Metric as PBMetric
+
+# pylint: enable=no-name-in-module
+
+
+class Metric(Enum):
+    """List of supported metrics."""
+
+    # Default value
+    UNSPECIFIED = PBMetric.METRIC_UNSPECIFIED
```

Comment:
I think the doc for each field needs to be copied from the proto specs.

Reply:
Happy to add them!

Reply:
The docs are the same as in the [proto](https://github.com/frequenz-floss/frequenz-api-common/blob/2a28ec452861cde9afb520cb6cb67f8a30abf8fb/proto/frequenz/api/common/v1/metrics/metric_sample.proto#L65), I've checked again. Or did you mean something different?

Reply:
Oh, sorry, I thought there was a docstring for each field. Looks good then.